### PR TITLE
Feature/myforms

### PR DESCRIPTION
**Pull request by thiessenp-cds**
# Aborting this PR and recreating - please do not spend time reviewing it (will delete when new PR created)
# Summary | Résumé

Adds MyForm feature to display a list of templates in cards. Using a tab interface, the cards can be filtered by draft, published and all. 

# Test instructions | Instructions pour tester la modification

Run the app and click the MyForms link in the top admin nav. If you have no templates, first create a few in the FormBuilder. Try clicking the different tabs "draft", "published", "all". The related templates should be displayed. Each card also has a menu that displays a number of actions (currently pointing to fake links).

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

Add action links to the card menu. 

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [x] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [x] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [x] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [x] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: components/myforms/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+          {t("card.lastEdited")}: {formatDate(date)}
+        </div>
+        <div className="flex items-center text-sm">
+          <span className="font-bold mr-1" aria-hidden="true">
+            ⋮
+          </span>
+          <MenuDropdown
+            id={id}
+            title={t("card.menu.more")}
+            items={[
```
Comment (thiessenp-cds) [inline components/myforms/Card/Card.tsx:63]: Planning to update the card links in a future commit

File: pages/myforms/[[...path]].tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          url={`/${i18n.language}/myforms/all`}
+          isActive={path === "all"}
+          id="tab-all"
+          tabpanelId="tabpanel-all"
+        >
+          {t("nav.all")}
+        </Tab>
+      </Tabs>
+
+      <div className="mb-6">
```
Comment (thiessenp-cds) [inline pages/myforms/[[...path]].tsx:82]: Tabpanels should really follow right after the Tabs. One solution would be to move the "create" link above or below the tabs & tabpanels and visually put the link in between. For now, I left the link as is

**Comment by timarney** [comment]
Seeing this error when trying to run look at this PR.


Screen Shot 2022-11-07 at 2 02 12 PM

**Comment by thiessenp-cds** [comment]
Oddly I think this PR lost a commit or two and also seems to have pulled in other changes. I'm going to close this commit and try recreating it.

**Pull request closed without merging**